docs/mllib-statistics.md

```diff
@@ -584,24 +299,24 @@ distribution `N(0, 1)`, and then map it to `N(1, 4)`.
 
 Refer to the [`RandomRDDs` Java docs](api/java/org/apache/spark/mllib/random/RandomRDDs) for details on the API.
 
-{% highlight java %}
-import org.apache.spark.SparkContext;
-import org.apache.spark.api.JavaDoubleRDD;
-import static org.apache.spark.mllib.random.RandomRDDs.*;
-
-JavaSparkContext jsc = ...
-
-// Generate a random double RDD that contains 1 million i.i.d. values drawn from the
-// standard normal distribution `N(0, 1)`, evenly distributed in 10 partitions.
-JavaDoubleRDD u = normalJavaRDD(jsc, 1000000L, 10);
-// Apply a transform to get a random double RDD following `N(1, 4)`.
-JavaDoubleRDD v = u.map(
-  new Function<Double, Double>() {
-    public Double call(Double x) {
-      return 1.0 + 2.0 * x;
-    }
-  });
-{% endhighlight %}
+-{% highlight java %}
+ -import org.apache.spark.SparkContext;
+ -import org.apache.spark.api.JavaDoubleRDD;
+ -import static org.apache.spark.mllib.random.RandomRDDs.*;
+ -
+ -JavaSparkContext jsc = ...
+ -
+ -// Generate a random double RDD that contains 1 million i.i.d. values drawn from the
+ -// standard normal distribution `N(0, 1)`, evenly distributed in 10 partitions.
+ -JavaDoubleRDD u = normalJavaRDD(jsc, 1000000L, 10);
+ -// Apply a transform to get a random double RDD following `N(1, 4)`.
+ -JavaDoubleRDD v = u.map(
+ -  new Function<Double, Double>() {
+ -    public Double call(Double x) {
+ -      return 1.0 + 2.0 * x;
+ -    }
+ -  });
+ -{% endhighlight %}
```

Comment:
you mean putting a comment here, explaining why didn't create a separate example file?